### PR TITLE
Upgrade signers to 1.0.19

### PR DESCRIPTION
**Pull request by siladu**
Includes vertx upgrade and azure key vault upgrade with breaking change to SECP256K1 curve.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Next Version
+### Breaking Changes
+- Upgrade to signers 1.0.18 removes support for deprecated SECP256K1 curve.
```
Comment (usmansaleem) [inline CHANGELOG.md:5]: suggestion: Upgrade to signers 1.0.18 removes support for deprecated SECP256K1 curve in Azure remote signing.

File: acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/SecpSigningAcceptanceTest.java [hunk 2]
```diff
@@ -113,35 +97,39 @@ public void signDataWithKeyFromHashicorp(@TempDir Path keyConfigDirectory) {
...
     final Bytes signature = verifyAndGetSignatureResponse(response);
-    verifySignature(signature);
+    verifySignature(signature, publicKeyHex);
   }
 
-  void verifySignature(final Bytes signature) {
-    final BigInteger privateKey = new BigInteger(1, Bytes.fromHexString(PRIVATE_KEY).toArray());
-    final BigInteger expectedPublicKey = publicKeyFromPrivate(privateKey);
+  void verifySignature(final Bytes signature, final String publicKeyHex) {
+    ECPublicKey expectedPublicKey =
```
Comment (usmansaleem) [inline acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/SecpSigningAcceptanceTest.java:125]: nit: `final ECPublicKey expectedPublicKey ... ` (using final keyword)

File: gradle/versions.gradle [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 dependencyManagement {
   dependencies {
-    dependency 'com.fasterxml.jackson.core:jackson-databind:2.11.2'
+    dependency 'com.fasterxml.jackson.core:jackson-databind:2.12.5'
     dependency 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.11.2'
```
Comment (usmansaleem) [inline gradle/versions.gradle:17]: does `com.fasterxml.jackson.dataformat:jackson-dataformat-yaml` version required to be upgraded as well?

**Review by usmansaleem — APPROVED** [review]
LGTM - minor suggestions.

File: gradle/versions.gradle (hunk 3 above)
Comment (jframe) [reply gradle/versions.gradle:17]: Would be good to use a dependency set here too to keep the jackson versions in sync

**Review by jframe — APPROVED** [review]

File: gradle/versions.gradle (hunk 3 above)
Comment (usmansaleem) [reply gradle/versions.gradle:17]: @jframe  They are two different groups unfortunately.

Comment (siladu) [reply gradle/versions.gradle:17]: AFAIK, the azure upgrade doesn't require dataformat-yaml to be upgraded (signers doesn't use it) Updating dataformat-yaml causes CommandlineParserTest failures which is starting to feel out of scope for this PR so I'm going to leave it as 2.11.2 unless there's any objections. I'd rather handle it in a 'upgrade web3signers deps' PR.

Comment (usmansaleem) [reply gradle/versions.gradle:17]: 👍🏼 .. lets keep it as it is then.

**Review by usmansaleem — APPROVED** [review]
LGTM

**Pull request merged**